tls/vibe/stream/openssl.d

```diff
@@ -1225,6 +1253,59 @@ final class OpenSSLContext : TLSContext {
 
 alias SSLState = ssl_st*;
 
+static if (haveKeylog) {
+	// use phobos, because concurrent writes are not possible with vibe.
+	import std.stdio;
+	private __gshared File keyfile;
+	/**
+	 * Use the SSLKEYLOGFILE environment variable to log the key.
+	 *
+	 * Whatever filename is stored in the SSLKEYLOGFILE environment variable
+	 * determines where the keys will be logged. Each key will be logged on a
+	 * separate line.
+	 *
+	 * This must be called on every OpenSSLContext you wish to log the key. If the
+	 * environment variable is not set, this does not log the key.
+	 */
+	void keylogOnEnvVar(OpenSSLContext context) {
```

Comment:
```suggestion
	void setSSLKEYLOGFILECallback(OpenSSLContext context) {	
```
```suggestion
	void setSSLKeyLogFileCallback(OpenSSLContext context) {	
```

Or something similar ? The environment variable name is well-known at this point (and was pioneered by cURL AFAIK, OpenSSL just copied it), so I think the first version is the least ambiguous, but not very in line with the common style.
Another option is to use `install` instead of `set`.